modules/ROOT/pages/amazon/amazon-s3-connector-config-topics.adoc

```diff
@@ -33,3 +33,12 @@ doc:id="DOC_ID" >
 
 [NOTE]
 The access key and secret key are not listed in the connector configuration. Instead, the correct values are used from the values specified in the settings.
+
+== Next Step
+
+You can try out the xref:amazon/amazon-s3-connector-examples.adoc[Example].
```

Comment:
Please change to Kim-type wording, such as: "After you understand how to configure a master key and provide credentials, you are ready to try the [Examples]."